pywt/_extensions/_swt.pyx

```diff
@@ -165,6 +165,7 @@ cpdef swt_axis(np.ndarray data, Wavelet wavelet, size_t level,
     cdef common.ArrayInfo data_info, output_info
     cdef np.ndarray cD, cA
     cdef size_t[::1] output_shape
+    cdef signed long[::1] strides_view
```

Comment:
It's crashing on 64-bit Windows in CI, I would guess because you're using `long` here which isn't going to be portable. This should be the actual type that numpy defines `strides` with, which IIRC is `intp`.